### PR TITLE
Added a simple example to show how to call v code from python

### PR DESCRIPTION
**Pull request by SaptakBhoumik**
I hope it will be helpful for python programmers

### CODE REVIEW

File: examples/call-v-from-python/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+A simple example to show how to call function written in v from python
```
Comment (spytheman) [inline examples/call-v-from-python/README.md:1]: Please rename the folder to `examples/call_v_from_python`. In general, dashes can create more problems compared to underscores, due to command line option parsing.

Comment (SaptakBhoumik) [reply examples/call-v-from-python/README.md]: Done

Comment (JalonSolov) [reply examples/call-v-from-python/README.md]: "... how to call **a** function..." just to be grammatical.

Comment (SaptakBhoumik) [reply examples/call-v-from-python/README.md]: Updated

**Pull request merged**